test.sh

```diff
@@ -0,0 +1,15 @@
+EXIT_STATE=0
+
+python -m unittest tests.development.test_base_component || EXIT_STATE=$?
+python -m unittest tests.development.test_component_loader || EXIT_STATE=$?
+python -m unittest tests.test_integration || EXIT_STATE=$?
+python -m unittest tests.test_resources || EXIT_STATE=$?
+python -m unittest tests.test_configs || EXIT_STATE=$?
+
+if [ $EXIT_STATE -ne 0 ]; then
+    echo "One or more tests failed"
+else
+    echo "All tests passed!"
+fi
+
+exit $EXIT_STATE
```

Comment:
Can we include the linting also ?

Reply:
Sure! I'll move linting in here too.

Reply:
linting into test.sh -> https://github.com/plotly/dash/pull/550/commits/2c9a3c892da5d1cf61435da5e79ff675bdff093a